include/adaflo/navier_stokes.h

```diff
@@ -46,6 +46,13 @@ template <int dim>
 class NavierStokes : public FlowBaseAlgorithm<dim>
 {
 public:
+  NavierStokes(const Mapping<dim> &                              mapping,
```

Comment:
Not perfect (this information is possibly inconsistent with the mapping of the external `MatrixFree` object) but at least allows externally the mapping class.